### PR TITLE
[CPU] {GRM, LSTM, RNN} Cells and Sequences migration on nGraph.

### PR DESCRIPTION
**Pull request by nshchego**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.h [hunk 1]
```diff
@@ -13,6 +12,19 @@
 
 namespace MKLDNNPlugin {
 
+struct PortMap {
+    // Data map rule
+    int from; /**< Index of exteral data from ins/outs fields of CNNLayer */
```
Comment (mandrono) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.h:17]:
```suggestion
    int from; /**< Index of external data from ins/outs fields of node*/
```

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.cpp [hunk 2]
```diff
@@ -186,52 +181,213 @@ class staticValueCheck : public PortChecker {
...
+    return numIterations;
+}
 
-    n_iter = getNumIteration(*ti);
-    sub_graph.CreateGraph(ti->body, ext_mng, weightCache);
+bool MKLDNNTensorIteratorNode::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        if (!one_of(op->get_type_info(),
+                ngraph::op::v0::TensorIterator::type_info)) {
+            errorMessage = "Unsupported TensorIterator operation.";
```
Comment (mandrono) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.cpp:266]: maybe ```Only opset1 TensorIterator operation is supported```

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.cpp [hunk 3]
```diff
@@ -186,52 +181,213 @@ class staticValueCheck : public PortChecker {
...
+    const std::shared_ptr<const ngraph::Function> body = tiOp->get_body();
+    sub_graph.CreateGraph(body, ext_mng, weightCache);
 
     // Try to detect inputs and outputs by indexes
-    const auto &in_map = sub_graph.GetInputNodes();
-    for (const auto &in_data : ti->body.inputs) {
-        if (in_data->getName() == "const_holder") continue;
+    const auto &inMap = sub_graph.GetInputNodesMap();
+    for (const auto &param : tiOp->get_body()->get_parameters()) {
+//        if (in_data->getName() == "const_holder") continue;
```
Comment (mandrono) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_tensoriterator_node.cpp:291]: remove

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp [hunk 4]
```diff
@@ -60,16 +60,8 @@ std::vector<std::string> disabledTestPatterns() {
 
         /* ********************************************************** TEMPORARILY DISABLED TESTS ********************************************************** */
         // shared SLT test
-        R"(.*GRUCellTest.*)",
-        R"(.*GRUSequenceTest.*)",
         R"(.*StaticShapeLoopTest.*)",
```
Comment (mandrono) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp:63]: also should be enabled

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**